rero_ils/modules/api.py

```diff
@@ -137,7 +138,12 @@ def create(cls, data, id_=None, delete_pid=False,
         if not id_:
             id_ = uuid4()
         cls.minter(id_, data)
-        record = super(IlsRecord, cls).create(data=data, id_=id_, **kwargs)
+        # We don't need to keep the PID if any validation error
+        try:
+            record = super(IlsRecord, cls).create(data=data, id_=id_, **kwargs)
+        except (RecordValidationError, ValidationError) as e:
+            db.session.rollback()
+            raise e
```

Comment:
Maybe `raise` will interrupt the import of fixtures. We have to be clear if we like to interrupt the import or if we have to intercept the exception and display a error message for example.

Reply:
Which fixtures do you talk about? Those from `./scripts/setup`?

I will check that.

Reply:
On my side, it seems to be OK.